manifests/params.pp

```diff
@@ -67,7 +67,7 @@
   $email_from = "no-reply@${::domain}"
   $email_enabled = false
 
-  $lazy_enabled = false
+  $manage_squid = true
```

Comment:
Seems like a bad user experience, but whatever we can go this route and flip the bit if a user of the module comes along and wants it flipped.